### PR TITLE
utils: add 'version' option to each util

### PR DESCRIPTION
**Pull request by rgetz**
Today, when debugging things - its sometimes tricky to tell which version of things are installed, since downstream distirbutions split iio-utils and libiio. This commit adds an new option "-V" or "--version" to things to print out the utility version, and the library version (which hopefully matches) seperately.

### CODE REVIEW

File: tests/iio_common.c
Comment (pcercuei) [inline tests/iio_common.c:450]: `libiio` -> `Libiio`? :)

Comment (rgetz) [reply tests/iio_common.c]: sure - I thought that you or Lars always left it lower case on purpose...

**Comment by pcercuei** [comment]
Can we hold this one until #959 is merged? Because you would have to document this new flag.

**Comment by pcercuei** [comment]
@rgetz #959 has been merged, could you add a commit to your PR that adds documentation for the --version option?

**Comment by rgetz** [comment]
yes - will do later tonight. You should have it your tomorrow am.

**Comment by rgetz** [comment]
updated.

File: man/iio_common_cmds.1
Comment (pcercuei) [inline man/iio_common_cmds.1:7]: nit - just missing a comma after -V

Comment (rgetz) [reply man/iio_common_cmds.1]: thanks - fixed

**Review by pcercuei — APPROVED** [review]

**Pull request merged**
